robottelo.properties.sample

```diff
@@ -132,6 +132,9 @@ ssh_key=
 # bridge=BRIDGE_VALUE
 # gateway=GATEWAY_VALUE
 
+# For OSCAP Testing
+# [oscap]
+# content_path=~/ssg-rhel6-ds.xml
 
```

Comment:
Why is this part of intrastructure?
Initially added the .xml file to robottelo code base , but that would bring in 44000+ lines of code to the robottelo code base. So decided to make this part of Infrastructure.